src/js/dataMapping/recipients/topCategories.js

```diff
@@ -7,7 +7,7 @@ export const recipientCategories = [
     'awarding_agency',
     'awarding_subagency',
     'federal_account',
-    'cfda',
+    'assistance_listings',
     'naics',
```

Comment:
The table is no longer working.  I would suspect there's an issue with the data mapping here.  I would try changing this back to cfda here and the next line that was changed to see if that fixes the problem.
![Screen Shot 2022-11-17 at 3 14 17 PM](https://user-images.githubusercontent.com/121010/202551122-c4270b00-7a3b-4ee8-98b1-04e19b66d879.png)

Reply:
Also the icon is missing

Reply:
also what is the path to this page as well. I think its fixed just need to verify